kubernetes/salt/config/master.d/salt-master.conf

```diff
@@ -3,8 +3,18 @@ auto_accept: True
 interface: 0.0.0.0
 
 event_return: mysql
-mysql.host: '127.0.0.1'
-mysql.user: 'root'
-mysql.pass: 'salt'
-mysql.db: 'velum_development'
-mysql.port: 3306
+
+mysql:
+  host: '127.0.0.1'
+  user: 'root'
+  pass: 'salt'
+  db: 'velum_development'
+  port: 3306
+
+ext_pillar:
+  - mysql:
+    - query: 'SELECT SUBSTRING_INDEX(SUBSTRING_INDEX(pillar, ".", 1), ".", -1) AS key1,
```

Comment:
Can we add a comment here? An example of a database record and a link to documentation would be nice ([docs](https://docs.saltstack.com/en/latest/ref/pillar/all/salt.pillar.mysql.html)?)